src/app/utils/field-validation/validators/common.ts

```diff
@@ -0,0 +1,12 @@
+import { left, right } from 'fp-ts/lib/Either'
+
+import { ResponseValidator } from 'src/types/field/utils/validation'
+import { ISingleAnswerResponse } from 'src/types/response'
+
+export const notEmptySingleAnswerResponse: ResponseValidator<ISingleAnswerResponse> = (
+  response,
+) => {
+  if (response.answer.trim().length === 0)
+    return left('notEmptySingleAnswerResponse')
```

Comment:
we're returning different things in different places as the content of `left`; sometimes it's an error message, sometimes it's the name of the containing function. Should we make this more consistent?

Reply:
ok